src/components/Body/Button.js

```diff
@@ -0,0 +1,17 @@
+import Button from 'react-bootstrap/Button';
+
+function AddToCartButton(props) {
```

Comment:
evo ideje kako da se otarasis ovog `props.` - destructuring:

```suggestion
function AddToCartButton({id, variant, handleCard}) {
```